.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,41 @@
+// For format details, see https://containers.dev/implementors/json_reference/
+{
+    "name": "Java Developer Container",
+    "build": {
+        "dockerfile": "../Dockerfile",
+        "target": "developer"
+    },
+    "remoteEnv": {
+        // Allow X11 apps to run inside the container
+        "DISPLAY": "${localEnv:DISPLAY}"
```

Comment:
ChannelFinder has no desktop application, so I don't think this is necessary.

Reply:
deleted